### DISCUSSION TITLE
Help for finding Pareto optimum in N-dimensional data set

### DISCUSSION THREAD

**Discussion by wolfig** [jMetal/jMetal#463] [General]
It would be nice to get some directions how to approach the problem of finding the Pareto optimum in a numeric (Double) data set consisting of N columns and M rows. I am confused, what the variables and what the objectives are in this kind of problem. Furthermore, I cannot give bounds for variables. Do you have an example in your code base? Does this idea of using jMetal make sense for this kind of problem?

Background to this: I am trying to find the optimum in a data set representing particle accelerator N device settings repeated M times with different setting values. To each of the M settings I have K instrument readings giving information about the quality of the data set of N settings (e.g. particle beam intensity, bean energy distribution width, ...). If I understand jMetals documentation correctly, I need to find a Pareto optimum of K objectives (= instrument readings) and the variables are the N device settings related to the readings.

**Comment by ajnebro** [comment]
Hi.
  I'm not sure to understand the problem. If you have a multi-objective problem with K objectives that means that it is formulated around K functions that have to optimize at the same time, and those functions are in conflict (i.e., improving one implies a worsening of the others).  Is that the case?

  Defining an encoding for matrices shouldn't be an issue. The point is that you need to define variation operators for those solutions, as crossover and mutation operators if you intend to use genetic algorithms.
